java/buildconf/ivy/ivy-suse.xml

```diff
@@ -51,7 +51,7 @@
         <dependency org="suse" name="jakarta-persistence-api" rev="2.2.2" />
         <dependency org="suse" name="java-saml" rev="2.4.0" />
         <dependency org="suse" name="java-saml-core" rev="2.4.0" />
-        <dependency org="suse" name="javassist" rev="3.23.1" />
+        <dependency org="suse" name="javassist" rev="3.29.0" />
```

Comment:
```suggestion
        <dependency org="suse" name="javassist" rev="3.29.2" />
```
SUSE:Maintenance:29380 updated the source to version 3.29.2. I didn't check the other changes but for this I spot the difference because I remembered the version released.

Reply:
Thanks, changed it so that we resolve `javassist` and `netty` from `Leap_sle` in order to get the latest versions.